static/js/compose_ui.js

```diff
@@ -205,11 +207,69 @@ export function handle_keydown(event, textarea) {
         type = "italic";
     } else if (key === "l" && event.shiftKey) {
         type = "link";
+    } else if (key === "enter" && !user_settings.enter_sends) {
```

Comment:
Hmm, I think we might need to take some care with how this interacts with the main `Enter` key handler that implements enter-sends.  In particular, if enter-sends is enabled, we probably want shift/ctrl+Enter to do this logic.